Fixes for incorrect encoding.

sticks2681 opened the pull request.

luciensadi commented:
Also, it looks like a lot of the changes here were permission changes (644->755), is that accurate?

sticks2681 commented:
I don't exactly know what happened there. I reverted that second commit that had deleted dos2unix.c and changed a lot of other files. The fixes for incorrect encoding should only touch the obj files 850, 852, 854, 856, 859.
